lib/Extension/CodeTransformExtra/Rpc/GenerateAccessorHandler.php

```diff
@@ -42,18 +58,72 @@ public function configure(Resolver $resolver)
 
     public function handle(array $arguments)
     {
-        $sourceCode = $this->generateAccessor->generateAccessor(
-            SourceCode::fromStringAndPath(
-                $arguments[self::PARAM_SOURCE],
-                $arguments[self::PARAM_PATH]
-            ),
-            $arguments[self::PARAM_OFFSET]
-        );
+        $class = $this->class($arguments[self::PARAM_SOURCE]);
+
+        $this->requireInput(ListInput::fromNameLabelChoices(
+            self::PARAM_NAMES,
+            sprintf('Properties from "%s"', $class->name()),
+            $this->propertiesChoices($class)
+        )->withMultiple(true));
+
+        if ($this->hasMissingArguments($arguments)) {
+            return $this->createInputCallback($arguments);
+        }
+
+        $originalSource = $arguments[self::PARAM_SOURCE];
+        $newSource = SourceCode::fromStringAndPath($originalSource, $arguments[self::PARAM_PATH]);
+
+        foreach ((array) $arguments[self::PARAM_NAMES] as $propertyName) {
+            $newSource = $this->generateAccessor->generate(
+                $newSource,
+                $propertyName,
+                $arguments[self::PARAM_OFFSET]
+            );
+        }
 
         return UpdateFileSourceResponse::fromPathOldAndNewSource(
-            $sourceCode->path(),
-            $arguments[self::PARAM_SOURCE],
-            (string) $sourceCode
+            $newSource->path(),
+            $originalSource,
+            (string) $newSource
         );
     }
+
+    /**
+     * @param string $source
+     *
+     * @return ReflectionClass
+     */
+    private function class($source)
```

Comment:
We don't add docblocks in this project unless they add value -- here the type hint should be `string` and the return type should be `ReflectionClass` (an exception might be if we want to declare that the return value is an array of `Foo` (e.g. `@return Foo[]`) or a collection `Foo` of `Bar` (`@return Foo<Bar>`).

Reply:
My mistake, I normally always type hint...